src/test-support/mocks/fs.ts

```diff
@@ -93,11 +93,11 @@ export const fs: FS = {
   '/meshes/:mesh/dataplanes/_overview': _21,
   '/meshes/:mesh/dataplanes/:name': _19,
   '/meshes/:mesh/dataplanes/:name/_overview': _22,
+  '/meshes/:mesh/dataplanes/:name/_rules': _128,
```

Comment:
Self-note: I had a script to auto generate this file originally, I must remember to pull that out and bring it over here.